.github/workflows/release.yml

```diff
@@ -101,8 +108,8 @@ jobs:
     ##########################################################################################################
     # Publish packages to nuget feed - will not do for now, since we should think about autopublishing first #
     ##########################################################################################################
-    - name: Push NuGet package to the (prod) feed
-      run: dotnet nuget push "**/*.nupkg"  --api-key ${{ secrets.feed_api_key }} --source https://www.myget.org/F/frends-tasks/api/v2/package --skip-duplicate
+    - name: Push NuGet package(s) to the prod feed
+      run: dotnet nuget push "**/*.nupkg"  --api-key ${{ secrets.feed_api_key }} --source ${{ env.azure_artifacts_feed_url }} --skip-duplicate
```

Comment:
I don't think it is a good idea to skip duplicates in production feed, since if for some reason a version which we are trying to push to the feed already exists then the release should fail.